apps/example-orchestrator-ui/components/Subscriptions/Subscriptions.tsx

```diff
@@ -55,6 +54,9 @@ const COLUMN_LABEL_NOTE = 'Note';
 
 const FIELD_NAME_INLINE_SUBSCRIPTION_DETAILS = 'inlineSubscriptionDetails';
 
+const defaultHiddenColumns: Array<keyof Subscription> = [PRODUCT_NAME];
```

Comment:
I see this notation Array<keyof  T> and [keyof T] both used. Would it be a good idea to standarise?

Reply:
Introduced a alias "TableColumnKeys<T>"